sockeye/score.py

```diff
@@ -73,13 +73,18 @@ def score(args: argparse.Namespace):
 
         sources = [args.source] + args.source_factors
         sources = [str(os.path.abspath(source)) for source in sources]
-        target = os.path.abspath(args.target)
+        targets = [args.target] + args.target_factors
+        targets = [str(os.path.abspath(target)) for target in targets]
+
+        check_condition(len(targets) == model.num_target_factors,
```

Comment:
The check needs to be moved up before creating the paths as currently one gets a `TypeError: expected str, bytes or os.PathLike object, not NoneType` when no target factors are specified.